mne/viz/tests/test_3d.py

```diff
@@ -528,6 +531,7 @@ def test_plot_volume_source_estimates(mode, stype, init_t, want_t,
 @testing.requires_testing_data
 @requires_dipy()
 @requires_nibabel()
+@requires_version('dipy', '0.10.1')
```

Comment:
why?

Reply:
If I remember well, one of the functions called checks for this condition, so the test was failling because this check was not fullfilled on my system.